### PR TITLE
Cloud-native e-book generation

### PR DESCRIPTION
**Pull request by mvelosop**
## Summary

Minor changes detected while generating the e-book.

- Fix mixed-up link
- Reduce solution explorer message
- Update pixel density for rbac-* images to 96 DPI, so they print correctly (no difference in web view)
- Other minor fixes

### CODE REVIEW

File: docs/architecture/cloud-native/azure-security.md [hunk 1]
```diff
@@ -259,7 +259,7 @@ While Cosmos DB doesn't provide for supplying customer encryption keys, there ha
...
+Azure has all the tools necessary to release a highly secure product. However, a chain is only as strong as its weakest link. If the applications deployed on top of Azure aren't developed with a proper security mindset and good security audits, then they become the weak link in the chain. There are many great static analysis tools, encryption libraries, and security practices that can be used to ensure that the software installed on Azure is as secure as Azure itself. [static analysis tools](https://www.whitesourcesoftware.com/), [encryption libraries](https://www.libressl.org/), and [security practices](https://azure.microsoft.com/resources/videos/red-vs-blue-internal-security-penetration-testing-of-microsoft-azure/), [LibreSSL](https://www.libressl.org/) and [Red vs. Blue - Internal security penetration testing of Microsoft Azure](https://azure.microsoft.com/resources/videos/red-vs-blue-internal-security-penetration-testing-of-microsoft-azure/) are examples of that, respectively.
```
Comment (mvelosop) [inline docs/architecture/cloud-native/azure-security.md]: Hi @ardalis, could you confirm if the "static analysis tools" link is what you intended? it confused the generator so I "fixed" it like this, but I'm not sure 😊

Comment (ardalis) [reply docs/architecture/cloud-native/azure-security.md]: Looks good, thanks

File: docs/architecture/cloud-native/identity-server.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ IdentityServer4 is open-source and free to use. You can add it to your applicati
 
 ## Configuration
 
-IdentityServer supports different kinds of protocols and social authentication providers that can be configured as part of each custom installation. This is typically done in the ASP.NET Core application's `Startup` class in the `ConfigureServices` method. The configuration involves specifying the supported protocols and the paths to the servers and endpoints that will be used. Figure 8-X shows an example configuration taken from the IdentityServer4 Quickstart UI project:
+IdentityServer supports different kinds of protocols and social authentication providers that can be configured as part of each custom installation. This is typically done in the ASP.NET Core application's `Startup` class in the `ConfigureServices` method. The configuration involves specifying the supported protocols and the paths to the servers and endpoints that will be used. Figure 8-2 shows an example configuration taken from the IdentityServer4 Quickstart UI project:
```
Comment (mairaw) [inline docs/architecture/cloud-native/identity-server.md]: line 87 also says 8-X

**Review by mairaw — APPROVED** [review]
LGTM. I've fixed an additional 8-X reference and will merge once build finishes.

**Pull request merged**